### PR TITLE
feat: add contact options for group

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes https://github.com/fossasia/open-event-frontend/issues/4662
requires https://github.com/fossasia/open-event-server/pull/8111

Screenshot from 2021-07-19 22-03-59
Screenshot from 2021-07-19 22-08-20

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/group/group-view.hbs [hunk 1]
```diff
@@ -39,6 +39,21 @@
...
+          <h2>{{t 'Get in touch'}}</h2>
+          <div>
+            {{#if (and @group.contactEmail @group.contactLink)}}
+              <a target="_blank" href="mailto:{{@group.contactEmail}}" class="ui red button mr-2" rel="noopener">{{t 'Email'}}</a>
+              <a target="_blank" href="{{@group.contactLink}}" class="ui red button ml-2" rel="noopener">{{t 'Contact'}}</a>
+            {{else if @group.contactEmail}}
+              <a target="_blank" href="mailto:{{@group.contactEmail}}" class="ui red button mr-2" rel="noopener">{{t 'Email'}}</a>
+            {{else if @group.contactLink}}
+              <a target="_blank" href="{{@group.contactLink}}" class="ui red button ml-2" rel="noopener">{{t 'Contact'}}</a>
+            {{/if}}
```
Comment (maze-runnar) [inline app/templates/components/forms/group/group-view.hbs:53]: this if will also work same. Am I mising something?

```suggestion
            {{if @group.contactEmail}}
              <a target="_blank" href="mailto:{{@group.contactEmail}}" class="ui red button mr-2" rel="noopener">{{t 'Email'}}</a>
              {{/if}}
            {{if @group.contactLink}}
              <a target="_blank" href="{{@group.contactLink}}" class="ui red button ml-2" rel="noopener">{{t 'Contact'}}</a>
            {{/if}}
```

**Review by maze-runnar — DISMISSED** [review]

**Comment by MeghalBisht** [comment]
Screenshot from 2021-07-22 22-47-52
Screenshot from 2021-07-22 22-48-00
Screenshot from 2021-07-22 22-48-04

**Comment by MeghalBisht** [comment]
Mails - 
Screenshot from 2021-07-22 22-50-42
Screenshot from 2021-07-22 22-50-47

**Comment by MeghalBisht** [comment]
@mariobehling Please check the above comments.

**Review by mariobehling — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**